.github/workflows/checkBranch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+branch=${GITHUB_HEAD_REF#refs/*/v}
+if ! [[ $branch =~ [a-z]{2,}\/[0-9]+-[A-Za-z-]+$ ]]
```

Comment:
acronym can have unlimited characters? I remember something about max 3 characters, but it's possible that this requirement comes from another project 😄

Reply:
Yes, it should be exactly 3 characters. I changed it. 😄